### PR TITLE
Update Set-CsPhoneNumberAssignment.md

### PR DESCRIPTION
**Pull request by JensTrier**
Add again support for DR numbers with extension for resource accounts

### CODE REVIEW

**Comment by JensTrier** [comment]
@JohanFreelancer9 I added another example to the PS page and removed a note. Do you mind reviewing it again? Sorry about the extra work.

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -29,16 +29,10 @@ Set-CsPhoneNumberAssignment -Identity <String> -EnterpriseVoiceEnabled <Boolean>
 
 ## DESCRIPTION
 This cmdlet assigns a phone number to a user or resource account. When you assign a phone number the EnterpriseVoiceEnabled flag is automatically set to True.
+You can also assign a location to a phone number when you assign the phone number to a user.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:32]:
```suggestion

You can also assign a location to a phone number when you assign the phone number to a user.

```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 2]
```diff
@@ -84,6 +78,19 @@ Try { Set-CsPhoneNumberAssignment -Identity user5@contoso.com -PhoneNumber "+142
...
+$TempUser = "tempuser@contoso.com"
+$OldLoc=Get-CsOnlineLisLocation -City Vancouver
+$NewLoc=Get-CsOnlineLisLocation -City Seattle
+$Numbers=Get-CsPhoneNumberAssignment -LocationId $OldLoc.LocationId -PstnAssignmentStatus Unassigned -NumberType CallingPlan -CapabilitiesContain UserAssignment
+foreach ($No in $Numbers) {
+	Set-CsPhoneNumberAssignment -Identity $TempUser -PhoneNumberType CallingPlan -PhoneNumber $No.TelephoneNumber -LocationId $NewLoc.LocationId
+	Remove-CsPhoneNumberAssignment -Identity $TempUser -PhoneNumberType CallingPlan -PhoneNumber $No.TelephoneNumber
+}
+```
+This example shows how to change the location for unassigned Calling Plan subscriber phone numbers by looping through all the phone numbers and for each phone number temporarily assigning it with the new loction to a user and then unassign the phone number again from the user.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:92]:
```suggestion
This example shows how to change the location for unassigned Calling Plan subscriber phone numbers by looping through all the phone numbers, assigning each phone number temporarily with the new location to a user, and then unassigning the phone number again from the user.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 I added another example to the PS page and removed a note. Do you mind reviewing it again? Sorry about the extra work.

No problem at all. Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**